src/mrpro/phantoms/coils.py

```diff
@@ -45,16 +44,16 @@ def birdcage_2d(
     y_co = repeat(y_co, 'y x -> coils y x', coils=1)
 
     c = repeat(torch.linspace(0, dim[0] - 1, dim[0]), 'coils -> coils y x', y=1, x=1)
-    coil_center_x = dim[2] * relative_radius * np.cos(c * (2 * torch.pi / dim[0]))
-    coil_center_y = dim[1] * relative_radius * np.sin(c * (2 * torch.pi / dim[0]))
+    coil_center_x = dim[2] * relative_radius * torch.cos(c * (2 * torch.pi / dim[0]))
```

Comment:
if coil_center_x is numpy array,
(y_co - coil_center_y) will no longer type check as torch.tensor,
which will result in a typing error with torch.sqrt not being called on a tensor